data/data/config.tf

```diff
@@ -20,6 +20,16 @@ EOF
 
 }
 
+variable "bootstrapping" {
```

Comment:
How are we setting this to "true" for the initial provisioning? Should we set the default value here?

Reply:
Note that UPI installations do not use any of these terraform files.

Reply:
So the alternative to setting the default here, is to set the bootstrapping variable explicitly at https://github.com/openshift/installer/blob/5c07d6ebfc793212418f7bc1ef840eec58579e8c/pkg/tfvars/tfvars.go#L40-L52.